core/assets/bundles/bundle_fr.properties

```
@@ -2182,10 +2182,10 @@ block.phase-wall.description = Ce mur est moins puissant qu'un mur en thorium, m
 block.phase-wall-large.description = Ce mur est moins puissant qu'un mur en thorium, mais il peut dévier les balles, sauf si elles sont trop puissantes.
 block.surge-wall.description = Le plus puissant bloc défensif.\nA une faible chance d'envoyer des éclairs vers les ennemis.
 block.surge-wall-large.description = Le plus puissant bloc défensif.\nA une faible chance d'envoyer des éclairs vers les ennemis.
-block.scrap-wall.description = Protects structures from enemy projectiles.
-block.scrap-wall-large.description = Protects structures from enemy projectiles.
-block.scrap-wall-huge.description = Protects structures from enemy projectiles.
-block.scrap-wall-gigantic.description = Protects structures from enemy projectiles.
+block.scrap-wall.description = Protège les structures des projectiles ennemies.
+block.scrap-wall-large.description = Protège les structures des projectiles ennemies.
+block.scrap-wall-huge.description = Protège les structures des projectiles ennemies.
+block.scrap-wall-gigantic.description = Protège les structures des projectiles ennemies.
```

```suggestion
block.scrap-wall.description = Protège les structures des projectiles ennemis.
block.scrap-wall-large.description = Protège les structures des projectiles ennemis.
block.scrap-wall-huge.description = Protège les structures des projectiles ennemis.
block.scrap-wall-gigantic.description = Protège les structures des projectiles ennemis.
```
Tu aimes la lettre "e" toi 🫵

---

En effet =)